### PR TITLE
Expand list of allowable string prefixes

### PR DESCRIPTION
**Pull request by alanhdu**
Closes https://github.com/google/yapf/issues/513

### CODE REVIEW

**Comment by alanhdu** [comment]
I'm not a fan of just listing all the allowable prefixes in the Python 3 case, but my regex-fu is weak and I'm wasn't sure of a better way to do it.

**Comment by gwelymernans** [comment]
There might be a more compact way to write it, but that's how it's specified in the Python docs, so let's just keep it the way you have it.

Could you update the `CHANGELOG` file and add a test please?

**Comment by alanhdu** [comment]
@gwelymernans I've added a unit test. Is that good enough, or do you prefer to also add an integration / regression test?

File: yapftests/format_token_test.py [hunk 1]
```diff
@@ -32,6 +33,16 @@ def testSimple(self):
     self.assertEqual("FormatToken(name=COMMENT, value=# A comment)", str(tok))
     self.assertTrue(tok.is_comment)
 
+  def testIsMultilineString(self):
```
Comment (gwelymernans) [inline yapftests/format_token_test.py]: This should go into `reformatter_python3_test.py`. It has support for PY3 stuff.

**Pull request merged**